fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1365,6 +1374,13 @@ ACTOR Future<Void> applyMetadataToCommittedTransactions(CommitBatchContext* self
 		ASSERT(false); // ChangeCoordinatorsRequest should always throw
 	}
 
+	if (pProxyCommitData->encryptMode.isEncryptionEnabled() && !extraDomainIds.empty()) {
```

Comment:
nit: add documentation here detailing the need to issue encryption-fetch here along with `getResolution`; essentially the same discussion we had in https://github.com/apple/foundationdb/pull/9308#issuecomment-1420224787

Also, given the EK fetch is inline with CP path, it is better to add latency profiling here to account for extra penalty added (if any) due to EK fetch; also a counter to track number of such occurrences. 

Thanks.